src/messenger-shortcuts.js

```diff
@@ -115,7 +139,11 @@ function focusSearchBar() {
 }
 
 function focusMessageInput() {
-  document.querySelector('div[role="main"] div[role="textbox"]').click();
+  targetNode = getByAttr('div', 'aria-label', 'Type a message...')
+  triggerMouseEvent (targetNode, "mouseover");
```

Comment:
Really? That's odd. It turns out I remembered wrong&mdash;the `click()` is part of the official DOM API, not jQuery. Weird that your browser doesn't support it?